README.rst

```diff
@@ -167,8 +167,8 @@ solution here. A primitive parser could abort as soon as evidence
 surfaces that the document is malformed. A more sophisticated parser
 could process the problematic structure so as to generate an error but
 give salve something well-formed. For instance if parsing
-``<foo></baz>``, such parser could emit an error on encountering
-``</baz>`` and replace the event that would be emitted for ``</baz>``
```

Comment:
I don't remember why I used `baz` rather than `bar` here. (Maybe I copied from a more complex example and did not feel like changing names.) At any rate, there's no reason to change it to `bar`. `baz` is a common metasyntactic variable:

https://en.wikipedia.org/wiki/Metasyntactic_variable#English